src/vue/backend/src/views/guided-setup/Api.vue

```diff
@@ -11,8 +11,8 @@
         </template>
         <template v-slot:controls>
             <APIKey
+                :disabled="formDisabled"
                 :error="error"
-                :loading="loading"
```

Comment:
You'll still want the loading animation in addition to being disabled, you can use the `loading` or the `formDisabled` prop for both 
```
:disabled="loading"
:loading="loading"
// Or

:disabled="formDisabled"
:loading="formDisabled"
```